### ISSUE TITLE
Evaluate rich for use with our CLI

### ISSUE THREAD

**Issue by ossareh** [strawberry-graphql/strawberry#1331]
As discussed here: https://github.com/strawberry-graphql/strawberry/pull/1321#discussion_r726507452

tldr:

- Emoji's do not work by default on Windows
- https://github.com/willmcgugan/rich provides a cross platform way of interacting with terminals
- Eval it, see if it helps with emoji on windows

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/strawberry-graphql) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/strawberry-graphql/strawberry/issues/1331">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/strawberry-graphql/strawberry/issues/1331/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/strawberry-graphql/strawberry/issues/1331/pledge.svg">
</picture>
</a>

**Comment by mtskillman** [comment]
using rich will not allow for emoji usage on Windows as the shell(s) on Windows do not support emojis: https://stackoverflow.com/questions/64940181/cannot-display-emojis-in-windows-powershell-or-wsl-linux-terminal-using-python

I have tried this in the strawberry/cli/commands/server.py
```
from rich import print
...
def server(schema, host, port, log_level, app_dir):
    ...
    print(f"Running strawberry on http://{host}:{port}/graphql :Strawberry:\n")
```

this results in this output in the Windows terminal:
![image](https://user-images.githubusercontent.com/55630669/166288942-b0ff8984-3ba7-45ea-bada-d743d183afde.png)

**Comment by patrick91** [comment]
we use rich now 😊
